### PR TITLE
Remove problem inspector from unit tests

### LINKED ISSUE #7: Remove problem inspector from unit tests
The unit tests block on the problem inspector.  Suggest refactoring the code so that the problem inspector is not launched as part of the unit tests.

### PR DESCRIPTION
**Pull request by mcfarljm**
Minor reorg so that problem inspector is not launched with unit tests, which blocks the tests.  Added `--no-problem-inspector` command line argument to disable the problem inspector, and update the unit tests to pass this option.  Also consolidate unit tests to single file and remove duplicate integration test.

Closes #7.

### CODE REVIEW

File: clustering.py [hunk 1]
```diff
@@ -49,7 +49,17 @@ def get_max_distance(coordinates):
     return max_distance
 
 
-def cluster_points(scattered_points, filename):
+def cluster_points(scattered_points, filename, problem_insepctor):
```
Comment (randomir) [inline clustering.py:52]: insepctor

**Review by randomir — APPROVED** [review]
LGTM

**Comment by mcfarljm** [comment]
Thanks for the review.  Fixed spelling error and rebased.  Should be ready to go.

**Review by randomir — APPROVED** [review]
